Failure to run on ARM/M2

The macOS binary (albeit compiled for x86) failed to run:

```
dyld[39000]: Library not loaded: /usr/local/opt/gettext/lib/libintl.8.dylib
  Referenced from: <560CCD2E-10B3-3F09-AC49-AB40D4C49112> /Users/mike/.basher/bash
  Reason: tried: '/usr/local/opt/gettext/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/gettext/lib/libintl.8.dylib' (no such file), '/usr/local/opt/gettext/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/usr/lib/libintl.8.dylib' (no such file, not in dyld cache)
2023/08/23 07:54:44 signal: abort trap
```

I tried building the application locally, and it failed, I think [because of the `cd /` lines](https://github.com/gliderlabs/herokuish/blob/77da1de18694eec3f366e7ccb14dcfdd28f11c26/Makefile#L131).

Here's a snippet of the failure:

```
mkdir -p $WORK/b001/exe/
cd .
/opt/homebrew/Cellar/go/1.19.4/libexec/pkg/tool/darwin_arm64/link -o $WORK/b001/exe/a.out -importcfg $WORK/b001/importcfg.link -buildmode=exe -buildid=4gMU95uRQO7Ou_CNylYe/pbGUFIjrUCCTdUFDFL9B/mN64kN_8bPaL_xR1oNRT/4gMU95uRQO7Ou_CNylYe -extld=clang $WORK/b001/_pkg_.a
/opt/homebrew/Cellar/go/1.19.4/libexec/pkg/tool/darwin_arm64/buildid -w $WORK/b001/exe/a.out # internal
mkdir -p /Users/mike/go/bin/
mv $WORK/b001/exe/a.out /Users/mike/go/bin/go-bindata
rm -r $WORK/b001/
go-bindata include
make: go-bindata: No such file or directory
make: *** [bindata.go] Error 1
```

I'm not familiar with go, so I wasn't sure where to go (ha!) from here.

--- a/README.md
+++ b/README.md
@@ -167,6 +167,14 @@ Mounting your local app source directory to `/tmp/app` and running `/bin/herokui
        1 example, 0 failures
 ```
 
+If you are on macOS, you'll want to explicitly set the platform:
+
+```shell
+$ docker run --platform linux/amd64 --rm -v /abs/app/path:/tmp/app gliderlabs/herokuish /bin/herokuish buildpack test
+```
+
+However, there is a risk of compatibility issues when running on a different platform than the one you are developing on. If you are getting strange compilation or segfaults, try running the build process on an x86 platform.
+
 #### Troubleshooting
 
 If you run into an issue and looking for more insight into what `herokuish` is doing, you can set the `$TRACE` environment variable.
@@ -196,6 +204,14 @@ $ docker run --rm -e TRACE=true -v /abs/app/path:/tmp/app gliderlabs/herokuish /
 ----->' Unable to select a buildpack
 + exit 1
 ```
+
+You can also set a custom buildpack:
+
+```shell
+docker run -e BUILDPACK_URL="https://github.com/custom/buildpack.git#with-a-branch" -e STACK=heroku-20 -e TRACE=true --rm -v ./:/tmp/app -it gliderlabs/herokuish /bin/herokuish test
+```
+
+Note that the underlying buildpacks will not trace their commands with `TRACE=true` is enabled. They need to independently set `set -x` in order to trace execution.
 
 ## Contributing
 

